tests/test_external_search.py

```diff
@@ -90,6 +90,7 @@
     EndToEndSearchTest,
 )
 
+RESEARCH = Term(audience=Classifier.AUDIENCE_RESEARCH)
```

Comment:
In TestExternalSearchWithWorks add a really boring research book about Alice in Wonderland. Around line 825, add a test that this book only shows up on a search for "alice" if there is an audience that explicitly includes AUDIENCE_RESEARCH. In particular, when there is no audience filter, the book shouldn't show up.

Reply:
Also, need an "all ages" book.